UPGRADE-3.x.md

```diff
@@ -65,6 +65,12 @@ All files under the ``Tests`` directory are now correctly handled as internal te
 You can't extend them anymore, because they are only loaded when running internal tests.
 More information can be found in the [composer docs](https://getcomposer.org/doc/04-schema.md#autoload-dev).
 
+### Exporter service and class
```

Comment:
Again on the wrong place. Please be careful.

A check bot is very necessary for this IMO.